mungegithub/mungers/publisher.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo $@
+if [ ! $# -eq 5 ]; then
+    echo "usage: publisher source_dir destination_dir source_url destination_url token"
+    exit 1
+fi
+SRC="${1}"
+DST="${2}"
+SRCURL="${3}"
+DSTURL="${4}"
+TOKEN="${5}"
+# set up github token
+rm -f ~/.netrc
+echo "machine github.com login ${TOKEN}" > ~/.netrc
```

Comment:
Can you add this as an arg to git commands instead of blowing away .netrc and storing the token like this?